mangadex/api.py

```diff
@@ -578,7 +583,7 @@ def login(self, username: str, password: str):
         ---------------
         `ApiError`
         """
-        self.__auth_handler(json_payload={"username": username, "password": password})
+        self.__auth_handler(json_payload={"username": username, "password": password, "client_id": client_id, "client_secret": client_secret})
```

Comment:
According to https://api.mangadex.org/docs/02-authentication/personal-clients/#authenticating, you need to add a "grant_type" section to the JSON payload, like this. I tried with and without this, and it worked well with but not without.
```suggestion
        self.__auth_handler(json_payload={"grant_type": "password", "username": username, "password": password, "client_id": client_id, "client_secret": client_secret})
```